app/components/meeting-form.hbs

```diff
@@ -138,13 +148,19 @@
             </span>
           </AuHeading>
           <AgendaManager @zittingId={{@zitting.id}} @onSave={{perform this.fetchTreatments}} />
-
-          {{!-- Treatment of agenda --}}
+          
+          {{!-- Start of meeting --}}
           <AuHeading id="sectionFour" @level="2" @skin="3" class="au-c-onboarding-wrapper au-u-margin-top-huge">
             {{t "meetingForm.fourthSectionTitle"}}
+          </AuHeading>
+          <ZittingTextDocumentContainer @zitting={{@zitting}} @readOnly={{false}} @type={{"ext:intro"}}/>
```

Comment:
Nitpicky, but string arguments don't need curlies: `@type="ext:intro"`

Reply:
And the readOnly is hardcoded again (and not used either it seems?) :smile:.

Reply:
Yeah I need to still implement that, not sure if oscars branch is finalized, wanna have it consistant with how he does it

Reply:
@Asergey91 fyi just merged oscars branch!